jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpGenerator.java

```diff
@@ -103,6 +97,16 @@ public void reset()
         _needCRLF = false;
     }
 
+    public int getMaxHeaderBytes()
+    {
+        return _maxHeaderBytes;
+    }
+
+    public void setMaxHeaderBytes(int maxHeaderBytes)
+    {
+        _maxHeaderBytes = maxHeaderBytes;
+    }
```

Comment:
This is kind of a duplicate in the server as we already have `HttpConfiguration.getResponseHeaderSize()`, and we have use overflowing of that buffer to enforce a header size.

So why are we duplicating this mechanism?  Can't the client just allocate a buffer of a specific size when it is asked for a HEADERs buffer?